libs/sdk-ui-filters/src/DateFilter/DateRangePicker/TimeInput.tsx

```diff
@@ -0,0 +1,105 @@
+// (C) 2025 GoodData Corporation
+
+import React, { useCallback, useState, ChangeEvent } from "react";
+import cx from "classnames";
+import moment, { Moment } from "moment/moment.js";
+import { useId } from "@gooddata/sdk-ui-kit";
+
+import { TIME_FORMAT } from "../constants/Platform.js";
+
+import { InputErrorMessage } from "./InputErrorMessage.js";
+import { buildAriaDescribedByValue } from "./utils.js";
+import { IInputAccessibilityConfig, ITimeInputErrorMessageTexts, ITime } from "./types.js";
+
+export interface ITimeInputProps {
+    value: ITime;
+    inputLabel: string;
+    onChange: (time: ITime) => void;
+    accessibilityConfig: IInputAccessibilityConfig;
+    isTimeOrderError?: boolean;
+    errorMessageTexts: ITimeInputErrorMessageTexts;
+    isMobile: boolean;
+}
+
+const padTo2Digits = (num: number) => String(num).padStart(2, "0");
+
+const stringifyTime = (time: ITime) =>
+    time === undefined ? undefined : `${padTo2Digits(time.hours)}:${padTo2Digits(time.minutes)}`;
+
+const asTime = (time: Moment | undefined): ITime =>
+    time === undefined || !time.isValid() ? undefined : { hours: time.hours(), minutes: time.minutes() };
+
+export const TimeInput: React.FC<ITimeInputProps> = ({
+    value,
+    inputLabel,
+    onChange,
+    accessibilityConfig,
+    isTimeOrderError = false,
```

Comment:
can these two props be replaced by single errorText?: string. If defined it means there is error. Only single message from errorMessageTexts is used here

Reply:
Fixed. Originally, there were more errors.